### PR TITLE
Feature/dockerfile

### PR DESCRIPTION
**Pull request by fliiiix**
first draft for docker stuff

### CODE REVIEW

File: backend/start.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+service nginx start
+uwsgi --http 127.0.0.1:5000 --wsgi-file app.py --callable app --need-app --die-on-term --uid www-data
```
Comment (randombenj) [inline backend/start.sh]: Do we want to create a systemd service for this in the future?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# building frontend
+FROM node:13.1 as build-deps
+COPY web/package.json web/yarn.lock ./
```
Comment (randombenj) [inline Dockerfile]: Why do you first copy only this files then the whole folder?

Comment (fliiiix) [reply Dockerfile]: fixed

File: backend/start.sh (hunk 1 above)
Comment (fliiiix) [reply backend/start.sh]: probably the questions is how well systemd services in docker work

**Pull request merged**